Updating Klayout DRC to work with new IHP run_drc.py

rjordans opened the pull request:
This updates the Klayout DRC for IHP PDK to use the new run_drc.py script.

The run_drc.py script doesn't have an argument to override the final report name so the script is updated to figure out the right name of the report file.  This could also have been done with a glob, but I didn't want to include another library for this.

mole99 commented:
Thanks for the PR!
And good catch with the `jobs`.

To me it's fine if you include another library as long as it's part of the Python standard library.
You could use `glob` or `pathlib` for that.

Let me know if you want to make that change or if you want to keep it as it is.

rjordans commented:
After reading my code again I changed my mind and switched to a glob, this now also checks if the resulting list can be indexed (in case no report was generated)

mole99 commented:
Thanks Roel! Can you do a quick formatting of the code using [`blue`](https://pypi.org/project/blue/)?
